front/src/RefboxSettings/index.js

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import ConfigDropdown from "./ConfigDropdown";
+import DropBall from "./DropBall";
+import InOutButton from "./InOutButton";
+import styled from "styled-components";
+
+/**
+ * Gives the settings part of the refbox:
+ * - Drop ball
+ * - Go in
+ * - Go out
+ * - Selectors (Match selector & Role / demo selector), see ConfigDropdown.js
+ *
+ * Author: G.W. van der Heijden
+ * Author: S.A. Tanja
+ * Author: T.T.P. Franken
+ *
+ * Props:
+ * - dbOnClick: a function on what to do when DropBall is pressed
```

Comment:
Most event handlers start with on, so these don't follow this convention. Why? You may also want to consolidate these handlers, see turtle for that.